Added contributing.md documentation. (#42)
fredrikhgrelland/vagrant-hashistack: docs/CONTRIBUTING.md

# Contributing to vagrant-hashistack

### Bugs or inconsistencies
If you find a bug or something that does not work according to docs, please file an issue.

> In case you spot a documentation error, or you could make an addition to help others; feel free to file a PR directly.

Reviewer: ```suggestion
In case you spot a documentation error, or there is an addition you can make to help others; feel free to file a PR directly.
```

### Would you like to see a change or add a feature?
Please file an issue before creating a PR.

> PRs are welcome if they are properly documented, including tests and corresponds to the [main goal](../README.md) of this project.

Reviewer: ```suggestion
PRs are welcome if they are properly documented, including tests, and corresponds to the [main goal](../README.md) of this project.
```

### Where are versions for the different tools declared?
You may change the version of tools in this [file](../ansible/group_vars/all/variables.yml). Remember to update the [changelog](../CHANGELOG.md) and label the PR as a release (major/minor/patch).

### Release process

> PRs labled with minor, major or patch will be built and tested if any of these codepaths are changed:

Reviewer: ```suggestion
PRs labeled with minor, major, or patch will be built and tested if any of these codepaths are changed:
```

- `ansible/**`
- `packer/**`
- `vagrant/**`
- `test/**`

> On successfull merge, the build process will run again, now including a release job. 

Reviewer: ```suggestion
On successful merge, the build process will run again, now including a release job. 
```
